docs/ReleaseNotes.md

```diff
@@ -21,6 +21,8 @@ The included licenses apply to the following files:
 
 Place release notes for the upcoming release below this line and remove this line upon naming this release.
 
+### Version 1.8.2502
+
```

Comment:
Should we do a survey for any additional notable changes that missed being mentioned here?

Reply:
If we intend to branch today, I think we should perhaps cherry pick a change later, instead of blocking on that now.